README.md

```diff
@@ -1,112 +1,3 @@
-  * You will see a dropdown button "Use workflow from"; ensure this says the default branch `main`: <img src="./images/run-workflow-from-main.png" alt="Run workflow from main branch" width="300"/>
-
-To complete repository setup, address all issues filed by this workflow.
-
-## Local development
-
-It can be helpful to build the GitHub Pages site locally to check that passing parameter values is working as expected.
-
-### Installing GitHub Pages dependencies locally
-
-Installing the dependencies for GitHub Pages is best done in a separate ruby environment, managed by [`rbenv`](https://github.com/rbenv/) and [Bundler](https://bundler.io)
-
-The following instructions were tested for installation on macOS, but installation on other systems should be similar.
-
-1. [Install `rbenv`](https://github.com/rbenv/rbenv#installation).
-The easiest, and recommended, installation is through [Homebrew](https://brew.sh/):
-```
-# install rbenv
-brew update
-brew install rbenv ruby-build
-```
-Alternatively, you can install with [`rbenv-installer`](https://github.com/rbenv/rbenv-installer#rbenv-installer) which will work even without Homebrew (though it uses Homebrew if you have it).
-
-2. Install `rbenv` for your shell.
-Since this will depend on which shell you are using, you will want to run `rbenv init` to find the correct command to add you your shell configuration file.
-Note that this command does _not_ actually perform the initialization: it only prints instructions for what line to add to which file!
-Be sure to add that line to the _end_ of the configuration file indicated.
-
-4. Once you have modified your shell configuration, start a new shell to initialize `rbenv`.
-
-5. Install a current stable ruby version
-  ```
-  # install and set up v3.1.3 of Ruby
-  rbenv install 3.1.3 && rbenv rehash
-  ```
-
-6. Activate your ruby version. Here you have a few options:
-
-   a. If you want to set the global ruby version that will be used wherever you use `ruby`, you can use
-   ```
-   rbenv global 3.1.3
-   ```
-
-   b. To only use this version of ruby within the repository only, navigate to the root of this repository and use:
-   ```
-   rbenv local 3.1.3
-   ```
-   This will create a `.ruby-version` file that will automatically activate this version of ruby whenever you are in this directory.
-
-7. Now you should be ready to install Bundler:
-```
-gem install bundler
-```
-
-8. Finally, run `bundle install` from the root of this repository, where the `Gemfile` is located.
-This will install all additional dependencies.
-
-### Running a local jekyll server
-
-Once all dependencies are installed, you should be able to start a local jekyll server with:
-
-```
-bundle exec jekyll serve
-```
+## 2024 June: Childhood Cancer Data Lab "Introduction to Single-Cell RNA-seq" Workshop
 
+Please find the workshop website at: https://alexslemonade.github.io/2024-june-training/
```

Comment:
This is not active yet, so I might hold off on updating the readme?

Reply:
I'll make sure the issue stays open. I'd like to get a draft schedule in before turning on pages I think?

Reply:
> I'll make sure the issue stays open. I'd like to get a draft schedule in before turning on pages I think?

Makes sense to me